source/Exception.h

```diff
@@ -32,7 +32,7 @@
 #include <string>
 #include "Type.h"
 
-#define D6_THROW(exceptionClass, message) std::cerr << __FILE__ << " " << __LINE__ << " " << std::string(message) << std::endl; throw exceptionClass(__FILE__, __LINE__, message)
+#define D6_THROW(exceptionClass, message) { std::cerr << __FILE__ << " " << __LINE__ << " " << std::string(message) << std::endl; throw exceptionClass(__FILE__, __LINE__, message); }
```

Comment:
Tohle jsem myslim pridaval kvuli tomu, aby se ty vyjimky vypisovaly do logu u headless/dedicated serveru

Reply:
Melo by se to prave vypsat v tom Main.cpp. Ale pridal jsem jen zavorky, kdyby to bylo nekde pouzite treba v ifu.